jaas-ljm/example.js

```diff
@@ -1,26 +1,36 @@
+const LJM_SCRIPT = 'ljmScript';
+const BASE_SOURCE = 'https://8x8.vc/libs/lib-jitsi-meet.min.js';
+const BASE_SOURCE_STAGE = 'https://stage.8x8.vc/libs/lib-jitsi-meet.min.js';
 const REGION_SHARD_MAPPING = {
     'default': 'default',
     'frankfurt': 'eu-central-1',
     'london': 'eu-west-2'
 };
+const INVALID_CLASS = 'is-invalid';
+
 let options;
 let roomName;
 let token;
+let releaseVersion;
+let useStage;
 
-function buildOptions(tenant, roomName) {
-    const regionElem = document.querySelector('#regionInput');
-    const selectedRegion = regionElem.value;
+function buildOptions(tenant, room, release) {
+    const selectedRegion = document.querySelector('#regionInput').value;
     const hasRegion = selectedRegion !== 'default';
-    const region =  hasRegion ? `${selectedRegion}.` : '';
+    const region = hasRegion ? `${selectedRegion}.` : '';
+    const releaseVersion = release ? `?release=release-${release}` : '';
+    const stage = useStage ? 'stage.' : ''
+
     return {
+
         // Connection
         hosts: {
-            domain: '8x8.vc',
-            muc: `conference.${tenant}.8x8.vc`,
-            focus: 'focus.8x8.vc'
+            domain: `${stage}8x8.vc`,
```

Comment:
@saghul, can you please confirm the staging domains?

Reply:
Looks good!